shared/macros-ansible.jinja

```diff
@@ -0,0 +1,54 @@
+{{#
+  A wrapper over the Ansible lineinfile module. This handles the most common
+  options for us. regex is optional and when blank, it won't be included in
+  the Ansible script; this allows arbitrary additions to files. new_line will
+  only be passed when state is present. with_items will be specified only if
+  non-empty, allowing for iterating through a variable of content (with the
+  appropriate macro-based path). register will be specified only if non-empty,
+  allowing for saving the output of this lineinfile module.
+
+  Note that all string-like parameters are single quoted in the YAML.
+#}}
+{{% macro ansible_simple_line_in_file(msg='', path='', regex='', new_line='', create='no', state='present', with_items='', register='') -%}}
+- name: "{{{ msg }}}"
```

Comment:
A small idea for improvement: In some of the rules, we use `- name: "{{{ rule_title }}}"` which means the Ansible task has the same name as the title of the rule it belongs to. We can make the `msg` parameter optional and if `msg` isn't present we can put the rule title to `name` field.

Reply:
@jan-cerny Is there a oneline for this in Jinja or do I need an `if`/`else` statement?

Reply:
@cipherboy There is a built-in `default` filter (http://jinja.pocoo.org/docs/2.10/templates/#list-of-builtin-filters), but the if/else statement is well readable.

Reply:
I had gone with `msg or rule_title` which appears to work as well.

Reply:
@cipherboy wow, it does!